src/darwin/Framework/CHIP/MTRControllerFactory.mm

```diff
@@ -257,8 +257,8 @@ - (BOOL)startup:(MTRControllerFactoryParams *)startupParams
 
         // Initialize device attestation verifier
         const Credentials::AttestationTrustStore * trustStore;
-        if (startupParams.paaCerts) {
-            _attestationTrustStoreBridge = new MTRAttestationTrustStoreBridge(startupParams.paaCerts);
+        if (startupParams.paaCerts && startupParams.cdCerts) {
```

Comment:
Should this be `&&`, or `||`?  That is, should it be OK to provide only one set of certs, or does it have to be both or neither?

Reply:
I did some changes such that the `DefaultAttestationVerifier` use the `kTestAttestationTrustStore` if `CHIP_ERROR_NOT_IMPLEMENTED` is used. As such I have updated this code to require one *or* the other, not both.